builder/iOSTealiumTest/TealiumHelper.swift

```diff
@@ -38,7 +38,7 @@ class TealiumHelper {
     func start() {
         let config = TealiumConfig(account: "tealiummobile",
                                    profile: "demo",
-                                   environment: "dev",
+                                   environment: "prod",
```

Comment:
Just curious the reason we switched this to prod?

Reply:
This is a breaking change but it was like this a few months ago and was changed without realising this was supposed to be called from outside too. So this really is just a revert back to remove the breaking change.

It was changed to prod so you and i could test the changes since on "dev" there was a short circuit on some conditions that caused us not to notice the bug in the first place